lnrpc/autopilotrpc/autopilot_server.go

```diff
@@ -41,6 +41,13 @@ var (
 			Entity: "info",
 			Action: "read",
 		}},
+		"/autopilotrpc.Autopilot/SetScores": {{
+			Entity: "onchain",
```

Comment:
Only actually needs on chain perms?

Reply:
Rationale was that you can impact what channel gets opened by hitting this API, not sure if that is considered off-chain... Similar rationale for the `ModifyStatus` RPC above.